.github/workflows/on-push-app-extensions.yml

```diff
@@ -0,0 +1,92 @@
+name: Pipeline Contentful App Extensions
+
+on:
+  push:
```

Comment:
Does this mean we will deploy to prod even on PR branches?

Reply:
Oh, that's a good point. At first I thought about deploying even in PR branches. The app id is unique so it would deploy to prod in contentful. But yeah, I think it's better to limit only to master